blocks/commerce-cart/commerce-cart.js

```diff
@@ -97,6 +97,39 @@ export default async function decorate(block) {
         Footer: (ctx) => {
           const giftOptions = document.createElement('div');
 
+          if (ctx.item?.itemType === 'ConfigurableCartItem') {
+            const editLinkContainer = document.createElement('div');
+            editLinkContainer.className = 'cart-item-edit-container';
+
+            const editButton = document.createElement('button');
+            editButton.className = 'cart-item-edit-link';
+            editButton.textContent = 'Edit';
```

Comment:
Is this string translatable? We might need to include in a json file, but could be a minor bug fix for future reference.